### PR TITLE
cloud_project_network_private and cloud_project_network_private_subnet

### PR DESCRIPTION
**Pull request by hugueslepesant**
Litle contribution to improve Terraform's provider documentation.

### CODE REVIEW

**Review by scraly — CHANGES_REQUESTED** [review]
Thanks!
I've done the review :)

File: website/docs/r/cloud_project_network_private.html.markdown [hunk 1]
```diff
@@ -54,3 +54,11 @@ The following attributes are exported:
 * `regions_status/status` - (Deprecated) The status of the network in the region.
 * `status` - the status of the network. should be normally set to 'ACTIVE'.
 * `type` - the type of the network. Either 'private' or 'public'. 
+
+## Import
+
+Private network in a public cloud project can be imported using the `project_id` and `network_id`, separated by "/" E.g.,
```
[the remark this answers is not in the document]
Comment (hugueslepesant) [reply website/docs/r/cloud_project_network_private.html.markdown]: ok

File: website/docs/r/cloud_project_network_private_subnet.html.markdown [hunk 2]
```diff
@@ -73,3 +73,11 @@ The following attributes are exported:
 * `ip_pools/dhcp` - DHCP enabled.
 * `ip_pools/end` - Last ip for this region.
 * `ip_pools/start` - First ip for this region.
+
+## Import
+
+Subnet in a private network of a public cloud project can be imported using the `project_id` , `network_id` and `subnet_id`, separated by "/" E.g.,
```
[the remark this answers is not in the document]
Comment (hugueslepesant) [reply website/docs/r/cloud_project_network_private_subnet.html.markdown]: ok

File: website/docs/r/vrack_cloudproject.markdown [hunk 3]
```diff
@@ -36,3 +36,11 @@ The following attributes are exported:
 
 * `service_name` - See Argument Reference above.
 * `project_id` - See Argument Reference above.
+
+## Import
+
+Attachment of a public cloud project and a VRack can be imported using the `project_id`, `service_name` and `vrach_id`, separated by "/" E.g.,
```
[the remark this answers is not in the document]
Comment (hugueslepesant) [reply website/docs/r/vrack_cloudproject.markdown]: ok

**Review by scraly — APPROVED** [review]

**Comment by scraly** [comment]
Thanks! :)

**Pull request merged**